src/root/components/map/home-map/explanation-bubble.js

```diff
@@ -38,10 +38,12 @@ class ExplanationBubble extends React.Component {
               <Translate stringId='explanationBubbleType'/>
             </label>
             <dl className='legend__dl legend__dl--colors'>
+              {/* FIXME: this should be  #f63440 */}
```

Comment:
@imohkay @batpad what's the best way update these legend entries to match the circle colours on the map? Should I just add new css classes?